test/cpp/src/498-cache-plain-printer.cc

```diff
@@ -24,26 +24,26 @@ TEST_CASE("An empty cache stat block prints all zeros") {
 TEST_CASE("Hits increment the hit and access counts") {
   auto num_hits = 255;
   auto [line_index, hit_type, expected_line] = GENERATE(as<std::tuple<std::size_t, access_type, std::string>>{},
-      std::tuple{1, access_type::LOAD, "test_cache LOAD         ACCESS:        255 HIT:        255 MISS:          0 MSHR_MERGE:          0"},
-      std::tuple{2, access_type::RFO, "test_cache RFO          ACCESS:        255 HIT:        255 MISS:          0 MSHR_MERGE:          0"},
-      std::tuple{3, access_type::PREFETCH, "test_cache PREFETCH     ACCESS:        255 HIT:        255 MISS:          0 MSHR_MERGE:          0"},
-      std::tuple{4, access_type::WRITE, "test_cache WRITE        ACCESS:        255 HIT:        255 MISS:          0 MSHR_MERGE:          0"},
-      std::tuple{5, access_type::TRANSLATION, "test_cache TRANSLATION  ACCESS:        255 HIT:        255 MISS:          0 MSHR_MERGE:          0"}
+      std::tuple{1, access_type::LOAD, "cpu0->test_cache LOAD         ACCESS:        255 HIT:        255 MISS:          0 MSHR_MERGE:          0"},
+      std::tuple{2, access_type::RFO, "cpu0->test_cache RFO          ACCESS:        255 HIT:        255 MISS:          0 MSHR_MERGE:          0"},
+      std::tuple{3, access_type::PREFETCH, "cpu0->test_cache PREFETCH     ACCESS:        255 HIT:        255 MISS:          0 MSHR_MERGE:          0"},
+      std::tuple{4, access_type::WRITE, "cpu0->test_cache WRITE        ACCESS:        255 HIT:        255 MISS:          0 MSHR_MERGE:          0"},
+      std::tuple{5, access_type::TRANSLATION, "cpu0->test_cache TRANSLATION  ACCESS:        255 HIT:        255 MISS:          0 MSHR_MERGE:          0"}
   );
 
   cache_stats given{};
   given.name = "test_cache";
   given.hits.set({hit_type, 0}, num_hits);
```

Comment:
```suggestion
  given.hits.set({hit_type, 0}, num_hits);
  given.hits.set({hit_type, 1}, num_hits*3);
```

Reply:
I have tried implementing them, but because plain printer only checks for cpus less than NUM_CPUS, so plain printer doesn't print those stats out for comparison.